Tasks/AzureCLIV1/azureclitask.ts

```diff
@@ -140,7 +140,7 @@ export class azureclitask {
     }
 
     private static throwIfError(resultOfToolExecution): void {
-        if (resultOfToolExecution.stderr) {
+        if (resultOfToolExecution.code != 0) {
```

Comment:
Perhaps the function should be throwIfFailed to make it clearer?

Reply:
Good call. Fixed.